onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,1047 @@
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_UINT8,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+
+    std::set<int> supported_types_gpu = {
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+}
+
+void Capability::populate_op_mode_supported() {
+
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Unsqueeze"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Squeeze"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Cast"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Gather"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Mul"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Sub"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Min"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Div"});
+  no_dimension_supported.push_back({V_2020_4,{"All"},"Floor"});
+  no_dimension_supported.push_back({V_2021_2,{"All"},"Where"});
+  no_dimension_supported.push_back({V_2021_2,{"All"},"Range"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"ArgMin"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Max"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Add"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Less"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Greater"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Clip"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Resize"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Equal"});
+
+
+  subgraph_supported.push_back({V_2020_4,{"All"},"Mul"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Transpose"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Unsqueeze"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Cast"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Concat"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Gather"});
+  subgraph_supported.push_back({V_2020_4,{"Myriad"},"Div"});
+  subgraph_supported.push_back({V_2020_4,{"Myriad"},"Sub"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Identity"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Div"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Sub"});
+    
+  
+  //populate unsupportedmode_t
+  {
+    unsupportedopmode_t obj = {{V_2020_4,V_2021_1,V_2021_2},  
+     [this](const Node* node, const Provider_InitializedTensorSet&) {
+      for (size_t i = 0; i < node->InputDefs().size(); i++) {
+        if (node->InputDefs()[i]->TypeAsProto()->tensor_type().elem_type() != ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT)
+          return true;
+      }
+      return false;
+    }
+    };
+    _confirmation_map.insert({"Abs", obj});
+  }
+  {
+     unsupportedopmode_t obj = {{V_2020_4},
```

Comment:
Please copy over the comments as well for all the conditions.

Reply:
Resolved